data/basejob_ui_schema.json

```
@@ -125,7 +125,7 @@
                             "label": "Output Log",
                             "info": "Output Log help"
                         },
-                        "on": "input"
+                        "on": "finput"
```

Does this need to be hyphenated too?

```suggestion
                        "on": "f-input"
```